src/Libraries/RevitNodes/GeometryConversion/ProtoToRevitCurve.cs

```diff
@@ -141,6 +141,13 @@ private static Autodesk.Revit.DB.Curve Convert(Autodesk.DesignScript.Geometry.Nu
                 crv.IsRational);
 
         }
+        private static Autodesk.Revit.DB.Curve CreateNurbSpline(IList<XYZ> controlPoints, IList<double> weights, IList<double> knots, int degree, bool closed, bool rational)
```

Comment:
Here is what the Obsolete message says:

`[Obsolete("This function will be obsolete in Revit 2017. Use NurbSpline.CreateCurve(degree, knots, controPoints, weights) instead")]
        public static NurbSpline Create(IList<XYZ> controlPoints, IList<double> weights, IList<double> knots, int degree, bool closed, bool rational);`